cmslitemetadata_to_redshift/cmslitemetadata_to_redshift.py

```diff
@@ -92,6 +92,7 @@ def clean_exit(code, message):
     dbtables_metadata = data['dbtables_metadata']
     nested_delim = data['nested_delim']
     columns = data['columns']
+    ddl_file = data['ddl']
```

Comment:
https://github.com/bcgov/GDX-Analytics-microservice/blob/1f96d2e846a3e6651f8092712f910aa4d842c6c2/s3_to_redshift/s3_to_redshift.py#L111

Reply:
changed this to something that will work better for the code as we dont want the script to run if this is not set:
`if 'sql_query' not in data:
        clean_exit(1, "bad configuration")
    ddl_file = data['sql_query']`
Let me know if this doesnt work!